pages/builds/docker_containerized_builds.md.erb

```diff
@@ -101,11 +101,56 @@ To allow buildkite-agent to use the Docker client you’ll need to ensure its us
 
 Even though the buildkite agent cleans up the images after each run, inevitably over time your Docker host can fill up with a lot of unused images.
 
-We recommend running [docker-gc](https://github.com/spotify/docker-gc) or similar tool on a daily basis.
+Docker has released the [system prune](https://docs.docker.com/engine/reference/commandline/system_prune) command that will remove all unused containers, networks, and images from your environment. We recommend running either `docker system prune` or using a similar tool like [docker-gc](https://github.com/spotify/docker-gc) on a daily basis.
 
 ## Docker without Docker Compose
 
-The Buildkite Agent also has support for running builds for pipelines that only need a single container and Dockerfile to run. To use plain Docker instead of Docker Compose, add the following environment variable in your build step configuration:
+The Buildkite Agent also has support for running builds for pipelines that only need a single container and Dockerfile to run. The [Docker Buildkite Plugin](https://github.com/buildkite-plugins/docker-buildkite-plugin) provides a Docker Compose file and a set of hooks to your pipeline.
+
+To use the Docker plugin, add the `plugins` attribute to your command step. In this example, the `yarn` commands will be run inside a Docker container using the node:7 Docker image:
+
+```
+steps:
+  - command: yarn install && yarn run test
+    plugins:
+      docker#v1.1.1:
+        image: "node:7"
+        workdir: /app
+```
+
+There are many configuration options available for the Docker plugin. For the complete list, see the readme for the [Docker Buildkite Plugin on Github](https://github.com/buildkite-plugins/docker-buildkite-plugin).
+
+## Creating Your Own Docker Infrastructure
+
+If your team has significant Docker experience you might find it worthwhile invoking your own runner scripts rather than using the simpler built-in Docker support.
+
+To do this see the [agent hooks](/docs/agent/v2/hooks) and [parallelizing builds](parallelizing-builds) documentation.
+
+## Using Docker Compose without plugins
```

Comment:
Maybe we should make it clear that this is deprecated? "Using docker compose via the deprecated built-in support" or something?

Reply:
It would be good to have a 'deprecated' warning block we could add in to any piece of docs... I'll try one out here and we can re use it elsewhere!

Reply:
It's a bit wordy, but I wasn't sure how else to give all the info:

![image](https://user-images.githubusercontent.com/2074469/38970658-16b96b8a-43da-11e8-8c1a-d06e9ca84afd.png)

I've put another one in the Docker without plugins bit too